velox/common/base/SimdUtil.h

```diff
@@ -125,21 +125,20 @@ struct Vectors<int64_t> {
   using CompareType = __m256i;
 
   // Value of compareResult(xx) where all lanes are
-  // true. compareResult produces a scalar word representing the truth
+  // true. compareBitMask produces a bit mask representing the truth
   // values for a SIMD compare of this type, e.g. compareEq. Note that
   // in AVX the compare result is a vector of the shape of the compare
   // operands and in AVX512 this is a bit mask. So compareResult is a
   // mask extracting load in AVX and no-op in AVX512.
-  static constexpr uint32_t kAllTrue = 0xffffffff;
+  static constexpr uint32_t kAllTrue = 0xf;
```

Comment:
Maybe add a comment explaining why we only set one byte to 'f' now (assuming because we only have a handful of lanes for int64)